model/project_ref_test.go

```diff
@@ -138,6 +152,20 @@ func TestFindMergedProjectRef(t *testing.T) {
 	assert.True(t, mergedProject.WorkstationConfig.ShouldGitClone())
 	assert.Len(t, mergedProject.WorkstationConfig.SetupCommands, 1)
 	assert.Equal(t, "random2", mergedProject.TaskAnnotationSettings.FileTicketWebhook.Endpoint)
+	assert.Len(t, mergedProject.ParsleyFilters, 2)
+
+	projectRef.ParsleyFilters = []ParsleyFilter{}
```

Comment:
I'd suggest making a separate test for this, or adding a comment, to make it clear this is testing the MergeParsleyFilters behavior.